packages/runner/src/lib/base.scss

```diff
@@ -8,10 +8,6 @@ body {
 
 .runner,
 .reporter-wrap footer {
-  * {
-    box-sizing: border-box;
-  }
```

Comment:
One of the css definitions that's trying to be matched 10,000+ times with 0 matches. I don't see removing this impacting anything.

![Screenshot 2025-07-09 at 11 02 00 AM](https://github.com/user-attachments/assets/936cabb9-6f22-426d-b035-f1507f4bdbd9)